stories/setup/webpack.config.js

```diff
@@ -0,0 +1,3 @@
+require('babel-register');
```

Comment:
The main reason I am trying to use babel here is so I can import getClientConfig. I can't seem to use any of those utils files/functions if it is not done this way (without having an es5 version).